Linux/flash

```diff
@@ -207,11 +212,8 @@ while true; do
     esac
   done
 
-  # paranoia sync before umounting
-  echo "Synchronizing Filesystems"
-  sudo sync; sudo sync; sudo sync
-
-  writable=$(grep "${disk}" /etc/mtab | grep -e "\( rw,\|,rw\)")
+  $(sudo test -w "${disk}")
```

Comment:
This does not work for me. Testing this in an Ubuntu VirtualBox VM always returns writable=0 regardless if the SD card is writable or not.
Did you have problems with the /etc/mtab lookup?